src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -146,111 +135,85 @@ private static async Task<Document> InlineTemporaryAsync(Document document, Vari
         {
             var workspace = document.Project.Solution.Workspace;
 
```

Comment:
this is the meat of where things changed. unfortunately, the diff is terrible. effectively teh logic is pretty simple:

1. determine the expression we want to inline.
1. find the reference locations to inline and inline there.
1. if we had no conflicts, remove the declaration
1. add warning annotations for things like side-effects or moving code into conditionally compiled methods